src/controllers/hotkeys/HotkeyController.cpp

```diff
@@ -524,6 +523,14 @@ void HotkeyController::resetToDefaults()
     this->loadHotkeys();
 }
 
+void HotkeyController::clearRemovedDefaults()
+{
+    // The "toggleLiveOnly" argument was removed 2024-08-04
```

Comment:
I think the user should be somehow notified of this change on update. This also does not account for people who rebound it. If you don't want a message box when starting the program for everybody. IMO it would be nice to notify people who changed the default setting, maybe with a message box or a new element in the hotkey settings:

![chatterino-manual-config-migration](https://github.com/user-attachments/assets/401442c3-ba6d-464e-a781-bb5ff315e23c)

This cannot be a thing merged after this PR because then we completely break users' configs from one nightly to another. I can see it being a PR merged before this one and this one being made dependent on that.

Reply:
Effort has been made to make this more apparent for users in [`ea642bb` (#5530)](https://github.com/Chatterino/chatterino2/pull/5530/commits/ea642bb19b77540211bc09d504807f36859e577e) and [`23339d1` (#5530)](https://github.com/Chatterino/chatterino2/pull/5530/commits/23339d1d128f41c260fc5fe605abb772ce3d7923).

It doesn't highlight the rows themselves since it's a lot of code that's very messy to update, I'd rather that be part of another PR.

It currently looks like this:
![2024-08-10-152840_915x600_scrot](https://github.com/user-attachments/assets/083a611f-0367-4fd6-8335-276bf810acea)

It's not pretty, but I think it gets the job done.
I chose yellow/the position because that's how it's done for duplicate Commands.